.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+<!--
+
+ PLEASE MAKE SURE NO SENSITIVE INFORMATION ABOUT YOUR
+ SYSTEM IS EXPOSED WHEN SUBMITTING FEEDBACK!
+
+ Repository Mirroring Tool (RMT) is an open-source project and we can make it
+ even better with your help and feedback.
+
+ When adding your feedback/bug please keep in mind:
+   - This is not a support channel. If you are having paid support from SUSE
+     business please go to https://scc.suse.com/support for support.
+   - When posting bugs/feedback here, there is no guarantee that the issue
+     will be handled immediately. Depending on the topic and the availability
+     of the developers, the issue maybe resolved in undefined time.
+   - Make sure that you don't expose any personal or private information.
```

Comment:
I would maybe start this paragraph with sth like this:
`Keep in mind that your feedback and all communication around it will be publically accessable by anyone. So make sure ...`

Please rephrase it at will / with Will :gb:

Reply:
> publicly accessible